apphub/NLP/language_modeling/ptb.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Languge Modeling using LSTM on Penn Treebank\n",
+    "\n",
+    "Language Modeling is the development of models to predict the next word of the sequence given the words that precede it. In this notebook we will demonstrate how to do predict next word of a sequence using a LSTM. We will be using Penn Treebank dataset which contains 888K words for training, 70K for validation, and 79K for testing, with a vocabulary size of 10K."
```

Comment:
In this notebook we will demonstrate how to predict the next word of a sequence using an LSTM